tests/app/views/test_questionnaire.py

```diff
@@ -397,3 +397,63 @@ def test_remove_empty_household_members_middle_name_only_not_stored(self):
 
         for answer in unanswered:
             self.assertFalse(self.question_store.answer_store.exists(answer))
+
+    def test_page_title_should_be_survey_title_when_type_introduction(self):
+        # Given
+        g.schema_json = load_schema_file("test_final_confirmation.json")
+
+        # When
+        page_title = get_page_title_for_block(g.schema_json, Location('final-confirmation', 0, 'introduction'))
+
+        # Then
+        self.assertEqual(page_title, 'Final confirmation to submit')
+
+    def test_page_title_should_be_group_title_then_survey_title_when_type_interstitial(self):
+        # Given
```

Comment:
Would prefer 'and' rather than 'then' in the title, confused me as I thought it was a test pattern (Given, when, then) also it would be better to use schema_title as that is what the function is using